execution/block-partitioner/src/tests.rs

```diff
@@ -0,0 +1,356 @@
+// Copyright © Aptos Foundation
```

Comment:
i'm curious can V2 pass these test cases without any change to V2/test cases? V2 doesn't reorder by sender at the beginning and I was expecting some V1-specific tests will fail...

Reply:
I removed all sender specific tests.. all other tests are generic and they are passing on V2